components/navbar/CreateDropdown.vue

```diff
@@ -16,6 +16,17 @@
           </nuxt-link>
         </b-tooltip>
       </b-dropdown-item>
+      <b-dropdown-item has-link>
+        <b-tooltip
+          position="is-left"
+          :label="$t('createDropdown.massmint')"
+          class="navbar-item-tooltip">
+          <nuxt-link data-cy="massmint" :to="`/${urlPrefix}/massmint`">
+            {{ $t('multipleNFTS') }}
+          </nuxt-link>
+        </b-tooltip>
+      </b-dropdown-item>
```

Comment:
The option should be visible only on the `redesign` mode. Let's make the url parms to `?redesign=true` to see this nav item.

Reply:
Sure thing

Reply:
done